src/SmartComponents/Actions/ActionsOverview.js

```diff
@@ -48,30 +48,26 @@ class ActionsOverview extends Component {
         const {
             statsFetchStatus
         } = this.props;
+
+        const donutValues = this.state.category.map((value, key) => [ typeNames[key], value ]);
         const renderDonut = (donutValues) =>
             <Donut key='advisor-donut' values={ donutValues } link={ typeLink } totalLabel='issues' identifier='advisor-donut' withLegend/>;
-        let donutValues = [];
-        let SummaryChartItems = [];
-
-        this.state.severity.map((value, key) => {
-            SummaryChartItems.push(
-                <ConditionalLink
-                    key={ key }
-                    condition={ value }
-                    wrap={ children =>
-                        <Link to={ `/actions/${sevNames[key].toLowerCase()}-risk` }>
-                            { children }
-                        </Link>
-                    }>
-                    <SummaryChartItem
-                        name={ sevNames[key] }
-                        numIssues={ value }
-                        totalIssues={ this.state.total }/>
-                </ConditionalLink>
-            );
-        });
 
-        this.state.category.map((value, key) => donutValues.push([ typeNames[key], value ]));
+        let SummaryChartItems = this.state.severity.map((value, key) =>
```

Comment:
last thing, I believe we can const this....
```suggestion
        const SummaryChartItems = this.state.severity.map((value, key) =>
```

Reply:
Argh, thought I had done that originally...  One sec.